src/any_agent/config.py

```diff
@@ -102,7 +103,7 @@ class ServingConfig(BaseModel):
 
 
 class TracingConfig(BaseModel):
-    model_config = ConfigDict(extra="forbid")
+    model_config = ConfigDict(extra="forbid", arbitrary_types_allowed=True)
```

Comment:
We don't need to allow arbitrary types anymore, I don't think.

Reply:
Most probably not. The exporters will not come here anymore, but I still have this doubt. We have transient info to pass on, so should it be placed here, or rather in some other object, since pydantic models are more suited for serializable stuff????